admin/app/com/lucidchart/piezo/admin/views/job.scala.html

```diff
@@ -27,6 +27,9 @@ <h3 class="pull-left job-name">@currentJob.get.getKey.getGroup() &raquo; <span c
       <a id="runJob" class="piezo-button pull-right" data-toggle="tooltip" title="Run job" onclick="$('#runConfirm').show()">
         <span class="glyphicon glyphicon-play"></span>
       </a>
+      <a class="piezo-button pull-right" data-toggle="tooltip" title="Duplicate job" href="@piezoRoutes.Jobs.getDuplicateJob(currentJob.get.getKey.getGroup(), currentJob.get.getKey.getName())">
+        <span class="glyphicon glyphicon-plus-sign"></span>
```

Comment:
There is a glyphicon-duplicate icon that would make more sense here, but I think we need to update twitter bootstrap to use it.